pjsip/src/pjsua-lib/pjsua_acc.c

```diff
@@ -218,6 +218,18 @@ static void init_outbound_setting(pjsua_acc *acc)
     acc->rfc5626_status = OUTBOUND_WANTED;
 }
 
+/*
+ * Helper function to deinit registration info on account.
+ */
+static void regc_destroy(pjsua_acc *acc)
```

Comment:
since the name is regc_destroy(), put pjsip_regc_destroy() inside the function as well.